definitions/variable/variables.yaml

```
@@ -10550,3 +10550,423 @@
     description: Subsidies for energy infrastructure related to Electricity (transport
       and storage)
     unit: billion EUR2020/yr
+- Minerals|Aluminium|Wind:
+    description: Demand of Aluminium for Wind Power Plants
+    unit: Mt/yr
+- Minerals|Aluminium|PV:
+    description: Demand of Aluminium for PV Power Plants
+    unit: Mt/yr
+- Minerals|Aluminium|CSP:
+    description: Demand of Aluminium for CSP Plants
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Fuel Cells:
+    description: Demand of Aluminium for Stationary Fuel Cells
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (renewable):
+    description: Demand of Aluminium for Power Plants (renewable)
+    unit: Mt/yr
+- Minerals|Aluminium|Power Plants (fossil + nuclear):
+    description: Demand of Aluminium for Power Plants (fossil + nuclear)
+    unit: Mt/yr
+- Minerals|Aluminium|Stationary Batteries:
+    description: Demand of Aluminium for Stationary Batteries
+    unit: Mt/yr
+- Minerals|Aluminium|Electrolyers:
```

I guess there's a typo here? (and also in the other variables for this electrolyser demand)
```suggestion
- Minerals|Aluminium|Electrolysers:
```